Gemfile

```diff
@@ -23,6 +23,10 @@ end
 
 gem 'simplecov', '~> 0.8'
 
+if RUBY_VERSION < '2.0.0' || RUBY_ENGINE == 'java'
+  gem 'json', '< 2.0.0'
+end
```

Comment:
I'm still confused by this...when we install gems for cross repo testing, we run `bundle install` from that repo's directory, and if it needs `json < 2.0.0` it should specify it.